Jenkinsfile

```diff
@@ -1,5 +1,55 @@
 #!/usr/bin/env groovy
-common {
-  slackChannel = '#connect-eng'
-  upstreamProjects = 'confluentinc/common'
+
+/*
+Copied from jenkins-common/var/common.groovy
+todo: Merge this back into jenkins-common and call the new centralized function
+ */
+
+def defaultConfig = [
+        nodeLabel       : defaultNodeLabel(),
+        cron            : '@weekly',
+        testResultSpecs : ['junit': '**/*-reports/TEST-*.xml'],
+        slackChannel    : '#connect-eng',
+        upstreamProjects:'confluentinc/common'
+]
+
+def config = jobConfig({}, defaultConfig)
```

Comment:
The defaultConfig dict is meant to be used inside of jenkins-common only when creating a new type, like "common java" vs "common Python" jobs. 

For job specific configs, it is better to pass it as a code like:
``` 
def config = jobConfig {
    nodeLabel = ...
}
```

I guess you intended to merge this back into jenkins-common, so maybe it's ok. Either case, it's minor.

Reply:
This is a good point, but as you suggest I will keep it similar to the original so it's easy to merge back